honeycombio/resource_slo.go

```diff
@@ -86,12 +86,7 @@ func resourceSLOCreate(ctx context.Context, d *schema.ResourceData, meta interfa
 	}
 
 	dataset := d.Get("dataset").(string)
-	s, err := expandSLO(d)
```

Comment:
`unparam` was correct to call out that this function could never return an error (the signature was tying to match a bunch of the other `expand*` helpers)